packages/core/src/components/viewer-grid/viewer-grid.tsx

```diff
@@ -186,8 +190,9 @@ export class ViewerGrid {
   render() {
     return (
       <Host {...{ [this.mainSize]: true }}>
-        <div class="dso-map-panel">
+        <div aria-live="polite" class="dso-map-panel">
```

Comment:
Is dit de juiste plek voor de `aria-live="polite"` ? Ik vermoed dat ie OOK de inhoud van de knoppen gaat voorlezen.. Volgens mij zegt de scope van het issue ook dat de aria-live="polite" OP de `span.sr-only` moet.